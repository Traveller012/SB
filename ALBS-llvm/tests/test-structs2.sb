{ : int} main = [ 

    flt f;
    struct location l1;
    struct location l2;

    l1.latitude = 30.1;
    l1.longitude = 60.2;
    l2.latitude = 30.6;
    l2.longitude = 60.7;

    f = l1.latitude;

    print(f);
    print(l1.latitude);
    print(l1.longitude);  
    print(l2.longitude);     

    rtn 0;
]

struct location[

    flt latitude;
    flt longitude;

]
