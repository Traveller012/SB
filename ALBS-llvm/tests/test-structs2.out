30.100000
30.100000
60.200000
60.700000